src/puppetlabs/puppetdb/reports.clj

```diff
@@ -64,11 +66,11 @@
 
 (def report-v7-wireformat-schema
  (-> report-wireformat-schema
-     (dissoc :producer)))
+     (dissoc :producer :noop_pending)))
 
 (def report-v6-wireformat-schema
   (-> report-v7-wireformat-schema
-      (dissoc :catalog_uuid :cached_catalog_status :code_id)))
+      (dissoc :catalog_uuid :cached_catalog_status :code_id :noop_pending)))
```

Comment:
This is redundant since it starts with the v7 schema, which already has :noop_pending dissociated.